ackhandler/sent_packet_handler.go

```diff
@@ -380,6 +407,16 @@ func (h *sentPacketHandler) queuePacketForRetransmission(packetElement *PacketEl
 	h.stopWaitingManager.QueuedRetransmissionForPacketNumber(packet.PacketNumber)
 }
 
+func (h *sentPacketHandler) computeHandshakeTimeout() time.Duration {
+	duration := 2 * h.rttStats.SmoothedRTT()
+	if duration == 0 {
+		duration = 2 * defaultInitialRTT
+	}
+	duration = utils.MaxDuration(duration, minTPLTimeout)
```

Comment:
See above, I'd rather have SmoothedRTT be initialized with the 100ms. Also, why the factor of 2?

Reply:
https://quicwg.github.io/base-drafts/draft-ietf-quic-recovery.html#pseudocode